scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,101 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+|======================
+|OpenShift Version |etcd version|storage schema version
+|3.4 and earlier |2.x | v2
+|3.5 |3.x | v2
```

Comment:
Lets change this to `3.3 and earlier` and `3.4 and 3.5`. The 3.4 upgrade playbooks will upgrade you to at least etcd 3.0.

I don't think it's worth making it even more confusing, but we don't do anything to bound the version of etcd installed so even a clean install of 3.1 today would get etcd 3.1 or whatever the latest is at install time.